src/MCPServer/debian/control

```diff
@@ -15,7 +15,6 @@ Depends:
 	archivematica-dashboard(>=1.2.0),
 	dbconfig-common,
 	mysql-server,
-	python-pyinotify,
 	python-gearman,
 	python-mysqldb,
 	python-lxml,
```

Comment:
If we're installing lxml, mysql-python & gearman above, do we need the python-gearman, python-mysqldb, python-lxml still?